report/report.md

```diff
@@ -0,0 +1,106 @@
+# Milestone 1 Report
+ 
+## Task 1
+
+The MNIST dataset is a widely-used collection of handwritten digits for training machine learning models in image classification.
+It combines images from NIST's Special Databases 1 and 3: SD-1, featuring complex digits from high school students, and SD-3, with clearer digits from Census Bureau employees.
+
+**Is it a classification or regression problem?**
+
+The dataset presents a classification problem, specifically aimed at identifying and categorizing handwritten digits from 0 to 9 by analyzing patterns in the image pixel data.
+
+**What are the characteristics of the data set?**
+
+- Contains 60,000 training images and 10,000 testing images, each of which is a grayscale image of size 28x28 pixels.
+- The pixel values are stored as unsigned bytes (0 to 255), where 0 - white background, 255 -black foreground.
+- Data files are stored in a specific IDX file format, designed for vectors and multidimensional matrices.
+- Metadata such as magic numbers and sizes in each dimension are also included.
+
+## Task 4 
+**Explain in detail which steps were necessary to run the code**
+
+In our virtual machine, the python version needed to be clarified first:
+`sudo apt update`  
+`sudo apt install python3`  
+We decided to use python 3, as it is the currently recommended version.
+
+In our case, pip was not installed either:
+`sudo apt install python3-pip`  
+
```

Comment:
@merclehoczky  I think it's important to mention creating environment here

Reply:
You're right, I added it